src/punica.c

```diff
@@ -90,6 +91,51 @@ static void init_signals(void)
     }
 }
 
+static int plugins_load(basic_plugin_manager_t *plugin_manager,
+                        plugins_settings_t *plugins_settings)
+{
+    linked_list_entry_t *entry;
+    plugin_settings_t *plugin;
+    int load_is_successful;
```

Comment:
No need to use such verbose names in simple situations. Prefer `is_loaded`, `success`, `status` or similar.